src/css/components/_code-output.scss

```diff
@@ -25,6 +25,11 @@
     font-size: 0.9rem;
     display: block;
     padding: 14px;
+
+    @media (min-width: 640px) {
+      // Allow the code area to scroll past the floating action buttons.
+      margin-bottom: 200px;
```

Comment:
@jakearchibald I was thinking, maybe 200px is too much?

Reply:
Hm, it can't be much less before the code goes under the background-change icon.

Reply:
It looks like there's a lot of space in the screenshot above, but the text isn't all the way across on the final line.

Reply:
Yeah, that's what I meant. Not sure why this happens yet.